source/blog/2017-12-11-release-notes.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: ⛄December 2017 Release Notes - Version 17.12.0
```

Comment:
Think you should have a space between the emoji and the "D" in "December"?

Reply:
Good call! Changed :)